runtimes/eoapi/stac/eoapi/stac/client.py

```diff
@@ -529,6 +550,16 @@ async def item_collection(
                 },
             )
 
+        elif output_type == MimeTypes.parquet:
+            return Response(
+                create_parquet(item_collection),
+                media_type=MimeTypes.parquet,
+                headers={
+                    "Content-Disposition": "attachment;filename=items.parquet",
+                    **additional_headers,
+                },
+            )
```

Comment:
Q: what happens when we're using the `fields` extension? the feature collection won't be valid STAC items ...

Reply:
No issues from the **stacrs** side we don't do any validation. I think if you rip out the `geometry` it might blow up, though.